packages/std/src/results/mod.rs

```diff
@@ -18,4 +19,5 @@ pub use handle::{HandleResponse, HandleResult};
 pub use init::{InitResponse, InitResult};
 pub use migrate::{MigrateResponse, MigrateResult};
 pub use query::{QueryResponse, QueryResult};
+pub use system::{SystemResponse, SystemResult as SystemRes};
```

Comment:
Ahh, good to know `SystemResult` already exists. We should keep that in mind and maybe find a better name for "system".